raiden/messages.py

```diff
@@ -488,16 +492,41 @@ class MediatedTransfer(LockedTransfer):
 
     cmdid = messages.MEDIATEDTRANSFER
 
-    def __init__(self, identifier, nonce, token, transferred_amount, recipient,
-                 locksroot, lock, target, initiator, fee=0):
+    def __init__(
+            self,
+            identifier,
+            nonce,
+            token,
+            transferred_amount,
+            recipient,
+            locksroot,
+            lock,
+            target,
+            initiator,
+            fee=0):
 
-        if nonce >= 2 ** 64:
+        if 0 > nonce >= 2 ** 64:
```

Comment:
I'm not sure that the conditional expression here is correct:
```python
In [1]: nonce = 5

In [2]: 0 > nonce >= 12
Out[2]: False
```

or to rephrase that: I believe the conditional expression is incorrect, or isn't it?